packages/signer/src/authClient.ts

```diff
@@ -0,0 +1,88 @@
+import type { AlchemyTransport } from "@alchemy/common";
+import { Signer } from "./signer.js";
+import type {
+  CreateTekStamperFn,
+  CreateWebAuthnStamperFn,
+  TurnkeyTekStamper,
+} from "./types.js";
+
+type TekStamperAndPublicKey = {
+  stamper: TurnkeyTekStamper;
+  targetPublicKey: string;
+};
+
+export class AuthClient {
+  constructor(
+    private readonly transport: AlchemyTransport,
+    private readonly createTekStamper: CreateTekStamperFn,
+    private readonly createWebAuthnStamper: CreateWebAuthnStamperFn
+  ) {}
+
+  private tekStamperPromise: Promise<TekStamperAndPublicKey> | null = null;
+
+  public async sendAuthEmail(email: string): Promise<void> {
+    const { targetPublicKey } = await this.getTekStamper();
+    const otpId = await notImplemented(email, targetPublicKey);
```

Comment:
is the `otpId` going to be used to map inflight auth flows in the session manager?

Reply:
nvrm you basically answer this question in the comment on the next line. I assume the answer is yes